admin/README.md

```diff
@@ -0,0 +1,35 @@
+## Intro
+Admin tool allows us to dynamically change settings of the running node without a restart. It can be used to change log level, and turn on profiler etc.
+
+## Usage
+
+### List all commands
+```
+curl localhost:9002/admin/run_command -H 'Content-Type: application/json' -d '{"commandName": "list-commands"}'
+```
+
+### To change log level
+```
+curl localhost:9002/admin/run_command -H 'Content-Type: application/json' -d '{"commandName": "set-log-level", "data": "debug"}'
+```
+
+### To turn on profiler
+```
+curl localhost:9002/admin/run_command -H 'Content-Type: application/json' -d '{"commandName": "set-log-level", "data": "debug"}'
```

Comment:
Copy and paste typo? This line looks identical to the `change log level` line, or?